src/callbacks/liquidity/BaselineV2/BaselineAxisLaunch.sol

```diff
@@ -460,11 +480,18 @@ contract BaselineAxisLaunch is BaseCallback, Policy, Owned {
         // There should not be any dangling approvals left
         Transfer.approve(RESERVE, address(BPOOL), proceeds_);
 
-        // Add all of the proceeds to the Floor range
-        BPOOL.addReservesTo(Range.FLOOR, proceeds_);
+        // Add the configured percentage of the proceeds to the Floor range
+        uint256 floorReserves = proceeds_ * floorReservesPercent / ONE_HUNDRED_PERCENT;
+        BPOOL.addReservesTo(Range.FLOOR, floorReserves);
+
+        // Add the remainder of the proceeds to the Anchor range
+        BPOOL.addReservesTo(Range.ANCHOR, proceeds_ - floorReserves);
 
         // Add proportional liquidity to the Discovery range
-        BPOOL.addLiquidityTo(Range.DISCOVERY, BPOOL.getLiquidity(Range.FLOOR) * 11 / 10);
+        BPOOL.addLiquidityTo(
+            Range.DISCOVERY,
+            (BPOOL.getLiquidity(Range.FLOOR) + BPOOL.getLiquidity(Range.ANCHOR)) * 11 / 10
```

Comment:
@fullyallocated can you confirm that we should multiply the sum of the floor and anchor liquidity by 11 / 10 here. I think we got this from the `initializeProtocol.sol`  policy.

Reply:
Currently it's (11/10) * anchor liquidity, FYI: https://github.com/0xBaseline/baseline-v2/blob/60bed78b7bee28016321ddd8c590df6c61bae6e9/src/policies/BaselineInit.sol#L186

Reply:
Sounds like we should change it to just the anchor then?

Reply:
From indigo:
> Yes this is fine, it adjusts itself after the first sweep anyway.